src/dreyfus_util.erl

```diff
@@ -194,6 +195,12 @@ maybe_deny_index(DbName, GroupId, IndexName) ->
             ok
     end.
 
+get_design_docid(DDoc) ->
```

Comment:
Is there a reason why `#doc` record is not used here. If I read the code correctly, the following would work:

```
get_design_docid(#doc{id = <<"_design/", DesignName/binary>>}) ->
     DesignName.    
```

Reply:
No specific reason. I think that you give one simple and better way.